### DISCUSSION TITLE
Scope per accesso ad API /info

### DISCUSSION THREAD

**Discussion by picce** [fattureincloud/api#372] [Q&A] [answered]
Nella pagina di documentazione degli SCOPES (https://developers.fattureincloud.it/docs/basics/scopes), non trovo riferimenti allo SCOPE da utilizzare per accedere alle API disponibili all'endpoint `/info`.
Nello specifico ho provato ad accedere alla seguente :
```
GET /c/{company_id}/info/vat_types
```

Il risultato è il seguente:
```
403Forbidden
{
    "error": {
        "message": "No permission.",
        "code": "NO_PERMISSION"
    }
}
```

**Comment by valmoz** [comment, marked as answer]
Buongiorno @picce ,
per questa chiamata lo scope è _settings_.
Ci faccia sapere se riesce ad effettuare la chiamata.

Grazie
Mauro

**Reply by picce** [reply]
grazie
